From a676c46de0401c52fbcf65bf624b66b14508cada Mon Sep 17 00:00:00 2001
From: Lucas Pardue <lucaspardue.24.7@gmail.com>
Date: Tue, 24 Sep 2019 01:27:33 -0400
Subject: strcase: fix raw lowercasing the letter X

Casing mistake in Curl_raw_tolower 'X' wasn't lowercased as 'x' prior to
this change.

Follow-up to 0023fce which added the function several days ago.

Ref: https://github.com/curl/curl/pull/4401#discussion_r327396546

Closes https://github.com/curl/curl/pull/4408
---
 lib/strcase.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/strcase.c b/lib/strcase.c
index 82c9db031..c080a5b35 100644
--- a/lib/strcase.c
+++ b/lib/strcase.c
@@ -150,7 +150,7 @@ char Curl_raw_tolower(char in)
   case 'W':
     return 'w';
   case 'X':
-    return 'X';
+    return 'x';
   case 'Y':
     return 'y';
   case 'Z':
-- 
2.30.2

